packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/enableAutocompletion.st

```diff
@@ -0,0 +1,5 @@
+utilities
+enableAutocompletion
+
+	autocompletionPreference := RichTextEditorModel autocompletion.
```

Comment:
Why no accessors?

Reply:
not necessary

Reply:
I thought we did settle on always using them